components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -62,9 +62,11 @@ internal class MediaSessionServiceDelegate(
     }
 
     fun onDestroy() {
-        scope?.cancel()
-
         logger.debug("Service destroyed")
+
+        scope?.cancel()
+        stopAll()
+        shutdown()
```

Comment:
The `shutdown()` call seems unnecessary in `onDestroy()`? I also wonder why we need to call `stopAll()` here?

Reply:
Confirmed that shutdown is needed.  Without shutdown when service is destroyed there an issue with YouTube when resuming video playback again later.
More testing shows that stopping all media session is not needed.  I've removed the changes for stopall().  Thanks,